src/test/scala/ai/privado/languageEngine/go/tagger/sink/GoAPITaggerTest.scala

```diff
@@ -22,14 +22,276 @@
  */
 package ai.privado.languageEngine.go.tagger.sink
 
+import ai.privado.cache.RuleCache
 import ai.privado.languageEngine.go.GoTestBase
 import ai.privado.model.*
+import ai.privado.testfixtures.GoFrontendTestSuite
 import io.shiftleft.semanticcpg.language.*
 
-class GoAPITaggerTestCase1 extends GoTestBase {
+class GoAPITaggerTestCase1 extends GoFrontendTestSuite {
+
+  val ruleCache = new RuleCache()
+
+  val sourceRule = List(
```

Comment:
The source Rules are already part of Test Rules, use them

Reply:
okay right, we can use them.